core/server/utils/url.js

```diff
@@ -297,6 +297,12 @@ function urlFor(context, data, absolute) {
         if (data && data.cors) {
             urlPath = urlPath.replace(/^.*?:\/\//g, '//');
         }
+
+        // CASE: urlFor('home') needs to be returned without trailing slash.
```

Comment:
The comment here is hard to understand.
Suggestion
// CASE: there are cases where the home url is requested without trailing slash e.g. blog url (see LINK why)